File: python/lc27.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        if len(nums) < 1:
            return 0
        idx = 0
        while idx < len(nums) and nums[idx] != val:
            idx += 1
        for i in range(idx, len(nums)):
            if nums[i] != val:
                nums[idx] = nums[i]
                idx +=1
        return idx

File: cs/lc27.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        if (nums.Length < 1)
            return 0;
        int idx = 0;
        while(idx < nums.Length && nums[idx] != val)
        {
            idx++;
        }
        for(int i=idx;i< nums.Length;i++)
        {
            if (nums[i] != val)
            {
                nums[idx] = nums[i];
                idx++;
            }
        }
        return idx;
    }
}
